packages/test/mocha-test-setup/mocharc-common.js

```diff
@@ -78,7 +78,14 @@ function getFluidTestMochaConfig(packageDir, additionalRequiredModules, testRepo
 
 	if (process.env.FLUID_TEST_MULTIREPORT === "1") {
 		config["reporter"] = `mocha-multi-reporters`;
-		config["reporter-options"] = [`configFile=${path.join(__dirname, "test-config.json")}`];
+		// See https://www.npmjs.com/package/mocha-multi-reporters#cmroutput-option
+		const outputFilePrefix = testReportPrefix !== undefined ? `${testReportPrefix}-` : "";
+		config["reporter-options"] = [
+			`configFile=${path.join(
+				__dirname,
+				"test-config.json",
+			)},cmrOutput=xunit+output+${outputFilePrefix}:mocha-json-output-reporter+output+${outputFilePrefix}`,
+		];
```

Comment:
Should we add a console log for the reporter-options content  ?

Reply:
sure. I think that's more useful than the console.log line below anyway.